conanfile.py

```diff
@@ -150,9 +150,14 @@ def _generate_about_versions(self, location):
             conan_installs.append([dependency.ref.name,dependency.ref.version])
 
         #list of python installs
-        import pkg_resources
-        for package in pkg_resources.working_set:
-            python_installs.append([package.key, package.version])
+        python_ins_cmd = f""" python -c "import pkg_resources; import os; [open(os.path.join("'"{self.recipe_folder}"'", "'"info.txt"'"), "'"a"'").write(s.key+"'","'"+ s.version+"'";"'") for s in pkg_resources.working_set]" """
```

Comment:
```suggestion
        python_ins_cmd = f""" python -c "import pkg_resources; import os; [open(os.path.join("'"{self._base_dir}"'", "'"info.txt"'"), "'"a"'").write(s.key+"'","'"+ s.version+"'";"'") for s in pkg_resources.working_set]" """
```

Always try to avoid generating files in the source folder and if you can't get around it make sure that the generated file is added to `.gitignore`.

In this case you could use `self._base_dir` which will point to the `venv` in the development environment or `cura_inst` when installing.